src/AppEntry.tsx

```diff
@@ -1,17 +1,22 @@
 import React from 'react';
 import { BrowserRouter as Router } from 'react-router-dom';
 import { Provider } from 'react-redux';
-import { init } from './store';
+import { RegistryContext, init } from './store';
 import App from './App';
 import { getBaseName } from '@redhat-cloud-services/frontend-components-utilities/helpers';
 import logger from 'redux-logger';
 
-const AppEntry = () => (
-  <Provider store={init(...(process.env.NODE_ENV !== 'production' ? [logger] : [])).getStore()}>
-    <Router basename={getBaseName(window.location.pathname)}>
-      <App />
-    </Router>
-  </Provider>
-);
+const AppEntry = () => {
+  const registry = init(...(process.env.NODE_ENV !== 'production' ? [logger] : []));
+  return (
+    <RegistryContext.Provider value={{ getRegistry: () => registry }}>
```

Comment:
What is the purpose of the added RegistryContext?

Reply:
To access registry from InventoryPage component and then do some magic copied from tasks repo to make component work :smile: